corehq/apps/export/models/new.py

```diff
@@ -2434,6 +2440,14 @@ def _reorder_case_properties_from_data_dictionary(cls, domain, case_type, case_p
 
         return ordered_case_properties
 
+    @classmethod
+    def _reset_case_property_group_name(cls, schema):
+        for group_schema in schema.group_schemas:
```

Comment:
Nit: There's a lot of nesting happening in this loop. One level of nesting can be removed by rearranging the code as:
```
for group_schema in schema.group_schemas:
    if group_schema.path != MAIN_TABLE:
        continue
    for item in group_schema.items:
        if isinstance(item, ScalarItem):
            item.case_property_group_name = ''
```

This is completely personal preference however, and happy for the code to stay as is.